bluemira/radiation_transport/radiation_tools.py

```diff
@@ -0,0 +1,1109 @@
+                n_split = np.int(np.ceil(y_width / max_wall_len))
+
+                # evaluate normal_vector
+                normal_vector = Vector3D(p1y - p2y, 0.0, p2x - p1x).normalise()
+
+                # Split up the wall component if necessary
+                splitwall_x = np.linspace(p1x, p2x, num=n_split + 1)
+                splitwall_y = np.linspace(p1y, p2y, num=n_split + 1)
+
+                y_width = y_width / n_split
+
+                for k in np.arange(n_split):
+                    # evaluate the central point of the detector
+                    detector_center = Point3D(
+                        0.5 * (splitwall_x[k] + splitwall_x[k + 1]),
+                        0,
+                        0.5 * (splitwall_y[k] + splitwall_y[k + 1]),
+                    )
+
+                    # to populate it step by step
+                    wall_detectors = wall_detectors + [
+                        (
+                            (ctr),
+                            x_width,
+                            y_width,
+                            detector_center,
+                            normal_vector,
+                            y_vector,
+                        )
+                    ]
+
+                    ctr = ctr + 1
+
+            else:
+
+                # evaluate normal_vector
+                normal_vector = Vector3D(p1y - p2y, 0.0, p2x - p1x).normalise()
+                # normal_vector = (detector_center.vector_to(plasma_axis_3D)).normalise()
+                # inward pointing
+
+                # evaluate the central point of the detector
+                detector_center = Point3D(0.5 * (p1x + p2x), 0, 0.5 * (p1y + p2y))
+
+                # to populate it step by step
+                wall_detectors = wall_detectors + [
+                    ((ctr), x_width, y_width, detector_center, normal_vector, y_vector)
+                ]
+
+                if debug:
+                    plt.plot([p1x, p2x], [p1y, p2y], "k")
+                    plt.plot([p1x, p2x], [p1y, p2y], ".k")
+                    pc = detector_center
+                    pcn = pc + normal_vector * 0.05
+                    plt.plot([pc.x, pcn.x], [pc.z, pcn.z], "r")
+
+                ctr = ctr + 1
+
+    if debug:
+        plt.show()
+
+    return wall_detectors
+
+
+def plot_radiation_loads(radiation_function, wall_detectors, wall_loads, plot_title, fw_shape):
+    """
+    To plot the radiation on the wall as MW/m^2
+    """
+
+    min_r = min(fw_shape.x)
+    max_r = max(fw_shape.x)
+    min_z = min(fw_shape.z)
+    max_z = max(fw_shape.z)
+
+    t_r, _, t_z, t_samples = sample3d(
+        radiation_function, (min_r, max_r, 500), (0, 0, 1), (min_z, max_z, 1000)
+    )
+
+    # Plot the wall and radiation distribution
+    fig, ax = plt.subplots(figsize=(12, 6))
+
+    gs = plt.GridSpec(2, 2, top=0.93, wspace=0.23)
+
+    ax1 = plt.subplot(gs[0:2, 0])
+    ax1.imshow(
+        np.squeeze(t_samples).transpose() * 1.0e-6,
```

Comment:
this will fix the orientation of the equilibria but I dont think it is the root of the problem...
```suggestion
        np.squeeze(t_samples)[::-1].transptose() * 1.0e-6,
```